packages/core/src/components/ResponsiveList/ResponsiveList.tsx

```diff
@@ -58,7 +56,7 @@ const ResponsiveList = forwardRef(
     }, [children, index]);
 
     const hiddenChildren = useMemo(() => {
-      const childrenArray = React.Children.toArray(children);
+      const childrenArray = React.Children.toArray(children) as React.ReactElement[];
```

Comment:
Can it be a single element?

Reply:
Even if it's a single element it is converted into an array of 1 item